client/core/core.go

```diff
@@ -1432,14 +1576,38 @@ func (c *Core) dbTrackers(dc *dexConnection) (map[order.OrderID]*trackedTrade, e
 	cancelPreimages := make(map[order.OrderID]order.Preimage)
 	trackers := make(map[order.OrderID]*trackedTrade, len(dbOrders))
 	for _, dbOrder := range dbOrders {
-		oid := dbOrder.Order.ID()
+		ord, md := dbOrder.Order, dbOrder.MetaData
+		oid := ord.ID()
 		var preImg order.Preimage
-		copy(preImg[:], dbOrder.MetaData.Proof.Preimage)
-		if co, ok := dbOrder.Order.(*order.CancelOrder); ok {
+		copy(preImg[:], md.Proof.Preimage)
+		if co, ok := ord.(*order.CancelOrder); ok {
 			cancels[co.TargetOrderID] = co
 			cancelPreimages[co.TargetOrderID] = preImg
 		} else {
-			trackers[oid] = newTrackedTrade(dbOrder.Order, dc, preImg)
+			// Make sure we have the necessary wallets.
+			wallets, err := c.walletSet(dc, ord.Base(), ord.Quote(), ord.Trade().Sell)
+			if err != nil {
+				log.Errorf("wallet retrieval error for active order %s: %v", oid, err)
+				continue
+			}
+			coinIDs := ord.Trade().Coins
+			if len(md.ChangeCoin) != 0 {
+				coinIDs = []order.CoinID{md.ChangeCoin}
+			}
+			coins := make(asset.Coins, 0, len(coinIDs))
+			if len(coinIDs) > 0 {
+				byteIDs := make([]dex.Bytes, 0, len(coinIDs))
+				for _, cid := range coinIDs {
+					byteIDs = append(byteIDs, reByte(cid))
```

Comment:
Can't we do `byteIDs = append(byteIDs, []byte(cid))` since the underlying type of `order.CoinID` is `[]byte`?